redaxo/src/core/lib/packages/manager.php

```diff
@@ -704,7 +704,12 @@ public static function synchronizeWithFileSystem()
             }
             $plugins = self::readPackageFolder(rex_path::addon($addonName, 'plugins'));
             foreach (array_diff($registeredPlugins, $plugins) as $pluginName) {
-                $manager = rex_plugin_manager::factory(rex_plugin::get($addonName, $pluginName));
+                $plugin = rex_plugin::get($addonName, $pluginName);
+                if (!$plugin instanceof rex_plugin) {
```

Comment:
hmm vllt. stattdessen in https://github.com/redaxo/redaxo/blob/f9a8cfff0c5ca0d9bbf356568f3657b63172edaf/redaxo/src/core/lib/packages/plugins/manager.php#L15 den typehint auf `rex_plugin_interface` ändern und im `__construct` die exception werfen?

Reply:
Habe es über neue `require()`-Methoden gelöst. Bitte mal schauen, was du davon hältst.

Reply:
find ich gut. wir sollte noch in den phpdocs von `::get()` herausstellen dass ein null-objekt returned werden kann.. damit man den unterschied zu require sieht